Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM node:10-alpine as builder
 
 WORKDIR /tmp/nexus-web
 COPY . /tmp/nexus-web
-RUN npm ci && npm run build
+RUN npm ci && node --max-old-space-size=4096 `which npm` run build
```

Comment:
There should be a comment next to this command to explain why we use max-old-space

Reply:
Done!